### ISSUE TITLE
Crash or unable to open UI in Ardour 4.7 with LV2 calf

### ISSUE THREAD

**Issue by YruamaLairba** [calf-studio-gear/calf#85]
almost "copy paste" from the same issue in the ardour tracker

**Crashing example :** try to insert Calf Deesser in ardour mixer, it will crash with this error in the terminal :

```
ardour-4.7.0: lv2gui.cpp :313 : void* gui_instantiate(const _LV2UI_Descriptor*, const char*, const char*, LV2UI_Write_Function, LV2UI_Controller, void**, const LV2_Feature* const*): l'assertion « xml » a échoué.
Abandon
```

**Unable to open UI example :** try to insert Calf Crusher in ardour, it give the following error in the terminal when trying to open the UI :

```
suil error: Failed to instantiate UI <http://calf.sourceforge.net/plugins/gui/gtk2-gui> [^] in /usr/lib/lv2/calf.lv2/calflv2gui.so
```

**Additional information**
debian testing (jessie)

Ardour 4.7.0
"Cluster and Eno"
(built from revision 1:4.7~dfsg-1)
Intel 64-bit

calf-plugins 0.0.60-1~bpo8+1

**Comment by boomshop** [comment]
Is this still an issue with latest git? Doesn't crash on my systems with Ardour 4.7.

**Comment by YruamaLairba** [comment]
yes, it's still an issue,i tried with ardour 4.4 and ardour 5.3. but i think it's a problem due to a library in my system, since the problem seems not present in other linux distribution. My system is also not up to date, the problem may be fixed..

**Comment by boomshop** [comment]
Thanks for reporting back! Feel free to reopen the report as soon as you made sure everything else can be excluded and the problem still exists.

**Comment by AndrewGreen** [comment]
I had this same problem, using Debian testing/unstable packages (ardour 5.5.0 and calf 0.60). Tried compiling Ardour from source (version 5.8.625), but that didn't help.  I was getting the same suil error in the console. Finally, it was fixed when I compiled calf plugins from source (current latest from github).

Here are some more bugs that appear to be related:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=856122
http://tracker.ardour.org/view.php?id=6937

Thanks so much for your work on this!!!! :)
